packages/plugin-divider/web/src/getToolbarButtons.tsx

```diff
@@ -45,9 +45,10 @@ const getNodeStyleResolver = {
 
 const selectedNodeResolver = {
   id: 'selectedNode',
-  resolve: content => {
-    if (Array.isArray(content) && content.length > 0) {
-      return content[0];
+  resolve: (_, __, editor) => {
+    const node = (editor?.state?.selection as Record<string, string> | undefined)?.node;
```

Comment:
```suggestion
    const node = editor?.state?.selection?.node;
```
This TS declaration is meaningless and just make it hard to read, please remove it from all places

Reply:
without the ts declaration we get a compilation error since node is not on type `Selection<any>`

Reply:
nevermind ):